### ISSUE TITLE
As a user, I want to assess restaurants' accessibility by reviewing and rating the accessibility checklist.

### ISSUE THREAD

**Issue by Huanjin-Zhang** [gcivil-nyu-org/nyc-accessible-restaurant-advisor#13]
**User story**
- As a user, I want to assess restaurants' accessibility by reviewing and rating the accessibility checklist.

**Acceptance criteria**
- If users are people with disabilities, they should be able to review and rate the accessibility checklist.

**Definition of Done**
- Users can review and rate the restaurants' accessibility checklist.
